common/src/main/kotlin/com/cd18/common/util/Time.kt

```diff
@@ -0,0 +1,32 @@
+package com.cd18.common.util
+
+import java.time.LocalDate
+import java.time.LocalDateTime
+import java.time.LocalTime
+import java.time.format.DateTimeFormatter
+
+fun getCurrentTime(): LocalDateTime = LocalDateTime.now()
+
+fun getTodayDate(): LocalDate = LocalDate.now()
```

Comment:
확장함수 정말 유용히 쓰고 있는 것 같아보여 🤗
이건 알 것 같지만, 너무 타입이 명확해보인다면 생략해서 좀 더 간결히 쓸 수도 있어

```suggestion
fun getCurrentTime() = LocalDateTime.now()

fun getTodayDate() = LocalDate.now()
```